.github/workflows/ci.yaml

```diff
@@ -9,7 +9,6 @@ on:
   push:
     branches:
       - master
-      - "[kmn]*-devel"
```

Comment:
I see why you would you disable the workflow for noetic, but why do you not want to run this on k/m pushes anymore?
Even if the pull-requests will be tested independently I would also test pushes.

Reply:
This file _is part of_ the `master` branch. The kinetic-devel and melodic-devel branches have their own CI config.
Thus, the setting was non-sense from the very beginning. However, it allowed to easily compare/share the config between those branches :wink: